src/main/kotlin/uk/gov/dluhc/printapi/service/PrintRequestsFileProducer.kt

```diff
@@ -0,0 +1,132 @@
+            ICSVWriter.DEFAULT_LINE_END
+        )
+        writer.writeNext(
+            arrayOf(
+                "requestId",
+                "issuingAuthorityEn",
+                "issuingAuthorityCy",
+                "issueDate",
+                "suggestedExpiryDate",
+                "requestDateTime",
+                "cardFirstname",
+                "cardMiddlenames",
+                "cardSurname",
+                "cardVersion",
+                "cardNumber",
+                "certificateLanguage",
+                "certificateFormat",
+                "deliveryOption",
+                "photo",
+                "deliveryName",
+                "deliveryStreet",
+                "deliverypProperty",
+                "deliveryLocality",
+                "deliveryTown",
+                "deliveryArea",
+                "deliveryPostcode",
+                "eroNameEn",
+                "eroPhoneNumberEn",
+                "eroEmailAddressEn",
+                "eroWebsiteEn",
+                "eroDeliveryStreetEn",
+                "eroDeliveryPropertyEn",
+                "eroDeliveryLocalityEn",
+                "eroDeliveryTownEn",
+                "eroDeliveryAreaEn",
+                "eroDeliveryPostcodeEn",
+                "eroNameCy",
+                "eroPhoneNumberCy",
+                "eroEmailAddressCy",
+                "eroWebsiteCy",
+                "eroDeliveryStreetCy",
+                "eroDeliverypPropertyCy",
+                "eroDeliveryLocalityCy",
+                "eroDeliveryTownCy",
+                "erodDeliveryAreaCy",
+                "eroDeliveryPostcodeCy"
+            )
+        )
+        printRequests.forEach { entry -> writer.writeNext(toStringArray(entry)) }
+        osw.flush()
+    }
+
+    private fun toStringArray(printRequest: PrintRequest): Array<String> =
+        with(printRequest) {
+            arrayOf(
+                requestId.toString(),
+                issuingAuthorityEn,
+                issuingAuthorityCy,
+                issueDate.toString(),
+                suggestedExpiryDate,
+                requestDateTime.toInstant().atOffset(ZoneOffset.UTC).format(DATE_TIMESTAMP_FORMATTER),
+                cardFirstname,
+                Optional.ofNullable(cardMiddlenames).orElse(""),
+                cardSurname,
+                cardVersion,
+                cardNumber,
+                certificateLanguage.toString(),
+                certificateFormat.toString(),
+                deliveryOption.toString(),
+                photo,
+                deliveryName,
+                deliveryStreet,
+                Optional.ofNullable(deliverypProperty).orElse(""),
+                Optional.ofNullable(deliveryLocality).orElse(""),
+                Optional.ofNullable(deliveryTown).orElse(""),
+                Optional.ofNullable(deliveryArea).orElse(""),
+                Optional.ofNullable(deliveryPostcode).orElse(""),
+                eroNameEn,
```

Comment:
How about this approach, @Neil-Massey ?

                deliverypProperty ?: "",
                deliveryLocality ?: "",
                deliveryTown ?: "",
                deliveryArea ?: "",
                deliveryPostcode ?: "",